build.sbt

```diff
@@ -113,9 +113,11 @@ lazy val zincRoot: Project = (project in file(".")).
         uncommittedChanges.exists(_.nonEmpty)
       },
 
-      // Ignore "git.gitCurrentTags.value.isEmpty"
-      // i.e. allow not-on-tag commits to be considered isSnapshot := false
-      isSnapshot := git.gitUncommittedChanges.value,
```

Comment:
That's fine. Ideally we should find a way to get scripted to define precise, stable versions instead of using -SNAPSHOT. I changed this to make sbt/sbt scripted tests much faster.